city_metrix/layers/layer.py

```diff
@@ -91,31 +108,42 @@ def _zonal_stats_fishnet(self, stats_func):
         tile_funcs = get_stats_funcs(stats_func)
 
         # run zonal stats per data frame
+        print(f"Input covers too much area, splitting into {len(tile_gdfs)} tiles")
         tile_stats = pd.concat([
             self._zonal_stats_tile(tile_gdf, tile_funcs)
             for tile_gdf in tile_gdfs
         ])
 
         aggregated = tile_stats.groupby("zone").apply(_aggregate_stats, stats_func)
+        aggregated.name = stats_func
 
-        return aggregated
+        return aggregated.reset_index()
 
     def _zonal_stats_tile(self, tile_gdf, stats_func):
         bbox = tile_gdf.total_bounds
         aggregate_data = self.aggregate.get_data(bbox)
         mask_datum = [mask.get_data(bbox) for mask in self.masks]
+        layer_data = self.layer.get_data(bbox) if self.layer is not None else None
 
         # align to highest resolution raster, which should be the largest raster
         # since all are clipped to the extent
-        raster_data = [data for data in mask_datum + [aggregate_data] if isinstance(data, xr.DataArray)]
+        raster_data = [data for data in mask_datum + [aggregate_data] + [layer_data] if isinstance(data, xr.DataArray)]
         align_to = sorted(raster_data, key=lambda data: data.size, reverse=True).pop()
```

Comment:
Isn't this going to return the smallest since `pop()` gets the last item and you are sorting them in reverse order?